dbeam-core/src/main/java/com/spotify/dbeam/jobs/BenchJdbcAvroJob.java

```diff
@@ -100,7 +100,7 @@ private String tsvMetrics() {
                 c ->
                     Optional.of(this.metrics.get(i).get(c))
                         .orElse(0L).toString()).collect(tabJoining),
-            this.metrics.get(i).get("bytesWritten") / this.metrics.get(i).get("writeElapsedMs")
+            this.metrics.get(i).get("bytesWritten")
```

Comment:
Can you elaborate on why this change?

Reply:
for some reason (sometimes), executing the tests locally, the bench test is throwing an ArithmeticException: /by zero. have you seen this behavior ?if it is not intended to run locally i can revert the changes and skip this test

Reply:
This seems to be unrelated to the main change. Might be put into a separate PR.

Reply:
I see. Then maybe `if this.metrics.get(i).get("writeElapsedMs") == 0` then it should be 0. Otherwise just to the division.

Feel free to have this as a separate PR.

Reply:
Will take this change out and create another PR